packages/i18next/src/useChangeLanguage.ts

```diff
@@ -0,0 +1,10 @@
+import { useCallback } from "react";
+import { useI18nextPlugin } from "./useI18nextPlugin.ts";
+
+export function useChangeLanguage<T extends string>() {
+    const plugin = useI18nextPlugin();
+
+    return useCallback((language: T) => {
```

Comment:
can't you just return plugin.changeLanguage? instead of wrapping it in useCallback?

Reply:
This is that I did first. But when doing this the function lose it's closure (context) and it doesn't work.